admin/config-ui/fields/class-field-google-search-console-intro.php

```diff
@@ -16,22 +16,24 @@ class WPSEO_Config_Field_Google_Search_Console_Intro extends WPSEO_Config_Field
 	public function __construct() {
 		parent::__construct( 'googleSearchConsoleIntro', 'HTML' );
 
-		$html = sprintf(
-			/* translators: %1$s is the plugin name, %2$s is a link start tag to a Yoast help page, %3$s is the link closing tag. */
-			esc_html__(
-				'%1$s integrates with Google Search Console, a must-have tool for site owners.
- It provides you with information about the health of your site.
- Don\'t have a Google account or is your site not activated yet?
- Find out %2$show to connect Google Search Console to your site.%3$s',
-				'wordpress-seo'
-			),
-			'Yoast SEO',
-			'<a href="' . esc_url( WPSEO_Shortlinker::get( 'https://yoa.st/1ex' ) ) . '" target="_blank">',
-			'</a>'
-		);
+		$html =
+			sprintf(
+				/* translators: %1$s is the plugin name. */
+				esc_html__( '%1$s integrates with Google Search Console, a must-have tool for site owners.', 'wordpress-seo' ),
+				'Yoast SEO'
+			) . '<br>' .
+			__( 'It provides you with information about the health of your site.', 'wordpress-seo' ) . '<br>' .
```

Comment:
For translation purposes, this sentence cannot be separated from the sentence before it. In some languages, the way 'it' should be translated depends on what 'it' is referring to. We could add translator comments to make the context clear. However, because it's not very likely those two sentences will appear without each other, keeping them in one translatable string sounds like a better idea

Reply:
Commited https://github.com/pedro-mendonca/wordpress-seo/commit/e4a4bf7bac3d58808a8462d4fb86d4f6f37b90d4 to solve this review.